src/cohort/cohort_server.cc

```diff
@@ -0,0 +1,33 @@
+#include "src/cohort/cohort_server.h"
+
+#include "absl/status/statusor.h"
+#include "grpcpp/server_context.h"
+#include "src/proto/cohort.grpc.pb.h"
+#include "src/utils/grpc_utils.h"
+
+namespace cohort {
+
+using grpc::ServerContext;
+
+absl::Status CohortServer::StartWork(const PrepareTransactionRequest &request) {
+  // TODO(benjmarks22): Create a fiber and start the transaction.
+  results_by_transaction_id_[request.transaction_id()]
+      .mutable_pending_response();
```

Comment:
to make sure i understand correctly, database operations will be implemented in this txn right?

Reply:
Yes that's correct.